flake.nix

```diff
@@ -1,8 +1,29 @@
 {
   description = "Poetry2nix flake";
 
-  outputs = { self }:
+  inputs.flake-utils.url = "github:numtide/flake-utils";
+  inputs.nixpkgs.url = "github:NixOS/nixpkgs";
+
+  outputs = { self, nixpkgs, flake-utils }:
     {
       overlay = import ./overlay.nix;
-    };
+    } // (flake-utils.lib.eachDefaultSystem (system:
+      let
+        myNixpkgs = import nixpkgs {
+          inherit system;
+          overlays = [ self.overlay ];
+        };
+      in
+      rec {
+        packages = {
+          inherit (myNixpkgs) poetry;
+        };
+        defaultPackage = packages.poetry;
```

Comment:
Why is the default app poetry and not poetry2nix?

Reply:
poetry is already available in nixpkgs itself.

Reply:
this repo provides a newer one generally. I haven't had the need to run poetry2nix yet. feel free to change it in a follow up PR.

Reply:
solved.